source/known-issues.txt

```diff
@@ -0,0 +1,31 @@
+.. _mongodb-analyzer-known-issues:
+
+============
+Known Issues
+============
+
+.. default-domain:: mongodb
+
+We are consistently working to improve the {+product+}, but we have
+identified certain issues with its current functionality. If you
+identify an issue that is not on this page, see the
+:ref:`mongodb-analyzer-issues-and-help` page for
+instructions on how to get help or report the issue.
+
+LINQ Expression Analysis on macOS
+---------------------------------
+
+If you are using the {+product+} in Visual Studio for macOS, you must
+include the following ``using`` directive at the beginning of your
+program to see analysis of LINQ expressions:
+
+.. code-block:: csharp
+
+   using MongoDB.Driver.Linq;
+
+Error List Window on macOS
+--------------------------
+
+If you are using the {+product+} in Visual Studio for macOS, you will
+not be able to see the output from the {+product+} in the Error List
```

Comment:
[nit/suggestion] Can we make this more specific than "output" (i.e. directly mentioning Query API representations)? This might just be me not knowing anything about the Analyzer yet. I _do_ know Query API translations are mentioned a good amount, but unsure if that's a completely different thing here.

Reply:
thanks for pointing that out! I changed the wording to reflect that output includes (but is not limited to) translations. I also changed wording to reflect that VS for Mac doesn't even have the error list window. also changed in the error list panel admonition.